common/main/java/com/couchbase/lite/internal/exec/ClientTask.java

```diff
@@ -31,18 +31,14 @@
 
 /**
  * Synchronous safe execution of a client task.
- * Motto: Their failure is not our failure.
  * <p>
- * I think it is ok to permit this executor pool to get very large: it is used only
```

Comment:
Turns out the sentiment expressed in this comment is completely false.  Making the pool that large has no advantage and may cause an OOM creating all those threads.